### PR TITLE
Create what-is-a-network-switch.md

### LINKED ISSUE #31: ELI5 - What is a network switch

### PR DESCRIPTION
**Pull request by Eviber**
closes #31

### CODE REVIEW

File: docs/what-is-a-network-switch.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# What is a network switch
+
+A network switch connects devices within a network and forwards data packets to and from those devices.  
+Unlike a router, a switch only sends data to the single device it is intended for, not to networks of multiple devices.
+
+## Layer 2 and Layer 3 switches
+Switches can operate on the layer 2 (via MAC addresses) or the layer 3 (via IP). Though some can do both, most switches are layer 2 switches.
```
Comment (FirelightFlagboy) [inline docs/what-is-a-network-switch.md:7]:
```suggestion

Switches can operate on the layer 2 (via MAC addresses) or the layer 3 (via IP). Though some can do both, most switches are layer 2 switches.
```

File: docs/what-is-a-network-switch.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## The CAM table
+The way it can forward packets to the desired device is by keeping a table called CAM (Content Addressable Memory) table.
```
Comment (FirelightFlagboy) [inline docs/what-is-a-network-switch.md:10]:
```suggestion

The way it can forward packets to the desired device is by keeping a table called CAM (Content Addressable Memory) table.
```

**Review by FirelightFlagboy — CHANGES_REQUESTED** [review]
Do we want to add a section for *dump* and *smart* switches ?

**Comment by Eviber** [comment]
I added a section about unmanaged vs managed switches.

**Review by FirelightFlagboy — APPROVED** [review]

**Review by Ragarnoy — APPROVED** [review]
